src/Program.cs

```diff
@@ -1,8 +1,14 @@
 using MatchService.Infrastructure;
 using MatchService.RequestPipeline;
-
+using Npgsql; // Para NpgsqlConnection
+using System.Data; 
+using MatchService.Domain;
+using MatchService.Data.Concretes;
 var builder = WebApplication.CreateBuilder(args);
 
+builder.Services.AddSingleton<IDbConnection>((sp) =>
```

Comment:
It is not necessary to add this, the table is already created with the script without using Services.AddSingleton.